src/problem/npv_170.rs

```diff
@@ -0,0 +1,25 @@
+use std::fmt;
+
+use derive_new::new;
+use relative_path::RelativePathBuf;
+
+#[derive(Clone, new)]
+pub struct ByNamePackegPrefixedWithNumber {
+    #[new(into)]
+    package_name: String,
+    #[new(into)]
+    relative_package_dir: RelativePathBuf,
+}
+
+impl fmt::Display for ByNamePackegPrefixedWithNumber {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let Self {
+            package_name,
+            relative_package_dir,
+        } = self;
+        write!(
+            f,
+            r#"- {relative_package_dir}: Attribute `{package_name}` should not be number-prefixed. It is suggestet to `"`-wrap this name"#
```

Comment:
> suggestet

typo

> `"`-wrap this name

I had to read that a couple of times before realizing what was meant. I think I'd prefer if it said something like "Wrap the name in quotes." However, it wasn't too clear to me when reading https://github.com/NixOS/nixpkgs/pull/333281 if wrapping in quotes is preferred over underscore prefix.

Reply:
I'm not sure what the recommendation should be. This was a best guess. I'd happily take direction on what the warning should _explicitly_ be.

Reply:
Oh looking at this again, realized they're not talking about the `pname` but the directory name for the package (which is what this checks :joy:). Can you instead only recommend the `_` prefix?